src/backend/src/prisma/schema.prisma

```diff
@@ -239,7 +239,6 @@ model Work_Package {
   project            Project              @relation(fields: [projectId], references: [projectId])
   orderInProject     Int
   startDate          DateTime
-  progress           Int                  @default(0)
```

Comment:
you need to run `yarn prisma:migrate` so that it generates  a migration to do this. see: https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8749072/Software+Prisma+FAQ#yarn-prisma%3Amigrate